symbols.tex

```diff
@@ -88,30 +88,39 @@
   \fi           
 }
 
-\newcommand{\FormulaVariable}[2]{
+% \FormulaVariable[<mod alignment>]{<mod desc>}{<content>}
+\newcommand{\FormulaVariable}[3][c]{
   \operatorname{
-    \tikz[baseline] \path
-      node[draw, shape=rectangle, anchor=base, fill=white] (box) { #2 }
-      (box.south) node[below=-0.5ex] {\makebox[0pt][c]{\scriptsize \engschrift #1}};
+    \tikz[baseline, trim right=(box.east)] \path
+      node[draw, shape=rectangle, anchor=base, fill=white] (box) { #3 }
+      \ifnum\pdfstrcmp{#1}{c}=0
+      (box.south) node[below=-0.5ex] {\makebox[0pt][#1]{\scriptsize \engschrift #2}}
+      \fi
+      \ifnum\pdfstrcmp{#1}{r}=0
+      (box.south east) +(0.5em, 0em) node[below=-0.5ex] {\makebox[0pt][#1]{\scriptsize \engschrift #2}}
+      \fi
```

Comment:
Alignment `l` could be added in a similar fashion, but is not needed so far, because the FormulaVariables tend to be at the right-hand side of the boxes.